js/analyzer.js

```diff
@@ -199,11 +299,47 @@ PreProcessor.prototype.textFormatter = function(){
  * saves wordcount (all words) and wordcountNoTags (all words except those in tags) in the _store object
  */
 PreProcessor.prototype.wordcount = function(){
+    /*wordcounters*/
     this._store.wordcount = this._store.cleanText.split(' ').length;
     this._store.wordcountNoTags = this._store.cleanTextNoTags.split(' ').length;
+    /*sentencecounters*/
+    this._store.sentencecount = this._store.cleanText.split('.').length;
+    this._store.sentencecountNoTags = this._store.cleanTextNoTags.split('.').length;
+    /*syllablecounters*/
+    this._store.syllablecount = this.syllableCount(this._store.cleanTextNoTags);
 };
 
 /**
+ * counts the number of syllables in a textstring.
+ * @param textString
+ * @returns syllable count
+ */
+PreProcessor.prototype.syllableCount = function(textString) {
+    var count = 0;
+    var wordsArray = textString.split(/[^aeiouy]/g);
```

Comment:
I find `wordsArray` a bit confusing. You might want to call it `chunksArray` or `chunkedWordsArray` :stuck_out_tongue_winking_eye: